### PR TITLE
🛑 🎛 (requested to be blocked) PRTL-1174 switch links that use gene id in filter to use gene symbol

### PR DESCRIPTION
**Pull request by cheapsteak**

### CODE REVIEW

File: modules/node_modules/@ncigdc/containers/CancerDistributionTable.js [hunk 1]
```diff
@@ -99,7 +98,7 @@ const CancerDistributionTableComponent = compose(
           key={b.key}
           ssms={explore.ssms}
           filters={makeFilter([
-            { field: 'genes.gene_id', value: [geneId] },
+            { field: 'genes.symbol', value: [entityName] },
```
Comment (cy) [inline modules/node_modules/@ncigdc/containers/CancerDistributionTable.js]: I'm not sure that this change, it's the filter for fetching mutation counts for the gene in each project, not a link to the explore page and junjun's said there could be genes with multiple ids? Might change the counts.

Comment (cy) [inline modules/node_modules/@ncigdc/containers/CancerDistributionTable.js]: correcting that, "it's the two genes have the same symbol, the symbol is marked with a `*`"

Comment (cy) [inline modules/node_modules/@ncigdc/containers/CancerDistributionTable.js]: same with ProjectBreakdown and the other table filters, I think should only change the `ExploreLinks` and the filters on `push` 🤔

**Comment by cheapsteak** [comment]
PR has been blocked
https://oicr-gdc.slack.com/archives/C0H0726P7/p1494435092757308
>Just had a discussion with Junjun on the Gene symbol / Gene id search, it does not work.
>Can you please hold on PRTL-1174 (@chang) and PRTL-1159

**Pull request closed without merging**